### PR TITLE
Update index.ts

### PR DESCRIPTION
**Pull request by TRANTANKHOA**
set `assetHashType: AssetHashType.SOURCE`, see https://github.com/aws/aws-cdk/pull/12984

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -87,6 +87,7 @@ export class ECRDeployment extends CoreConstruct {
     const handler = new lambda.SingletonFunction(this, 'CustomResourceHandler', {
       uuid: this.renderSingletonUuid(props.memoryLimit),
       code: lambda.Code.fromAsset(path.join(__dirname, '../lambda'), {
+        assetHashType: AssetHashType.SOURCE, // see https://github.com/aws/aws-cdk/pull/12984
```
Comment (wchaws) [inline src/index.ts:90]: @TRANTANKHOA What will happen if I don't set this? I don't really get it.

Comment (TRANTANKHOA) [reply src/index.ts:90]: I find cdk keeps rebuilding the lambda image on repeated builds, which is similar to the issue mentioned. I think change the hashType should fix that

Comment (TRANTANKHOA) [reply src/index.ts:90]: btw, I've tried your stack and the go lambda failed for me with error `fork/exec /var/task/main: permission denied: PathError null`. Do you have a hint why that is?

Comment (wchaws) [reply src/index.ts:90]: Ok, I'll merge this PR firstly. And Please verify if this can solve your problem.  @TRANTANKHOA

Comment (wchaws) [reply src/index.ts:90]: @TRANTANKHOA Can you create a sample repo how you write your stack? And which region did u deploy?

**Pull request merged**